### PR TITLE
Feat/text utilities

### PR DESCRIPTION
**Pull request by Debarshi95**
PR to add Text utilites

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -27,6 +20,6 @@ li {
   display: block;
   padding: 0.25rem;
```
Comment (ritikaawasthy) [inline css/styles.css]: you could use px for padding here as the value is small, otherwise looks great

**Review by ritikaawasthy — COMMENTED** [review]
looks clean

File: css/styles.css (hunk 1 above)
Comment (Debarshi95) [reply css/styles.css]: Will update it. Thanks

File: lib/index.css [hunk 2]
```diff
@@ -39,3 +41,81 @@
...
+  font-weight: 600;
+  width: 100%;
+  display: inline-block;
+  padding: 1rem 0;
+  text-align: left;
+}
+
+#sidebar-nav a:hover {
+  background-color: #bdbdbd;
+}
```
Comment (dhimanaishwarya) [inline lib/index.css:101]: use color variables

**Review by dhimanaishwarya — APPROVED** [review]
rest looks good. Will be good to have a preview link

**Pull request merged**